_getting-started/contracts-and-workspaces.md

```diff
@@ -114,3 +114,22 @@ non-disruptive testing one can create dedicated workspaces for testing and
 production stages, both running near-identical integration flows. The workspaces
 will have different credentials, so the testing stage may be accessed by the
 client’s engineers, and production environment is customer-facing only.
+
+
+## Limited Workspaces
+
+Workspaces can have the `limited` or `full` type. `Limited` is intended for platform trial period. Workspaces with `limited` type have certain restrictions for integration flows:
+
+- Integration flows in `limited` workspaces are restricted by work time. This time is defined for each installation with an environment variable.
```

Comment:
I would mention here explicitly what this restriction is, namely they are stopped after a pre-configured time. For example 2 hours after start time.

Reply:
done